lib/kiev/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Kiev
-  VERSION = "4.8.3"
+  VERSION = "4.8.4"
```

Comment:
is it a patch?

Reply:
it is a backwards compatible extension. I am not aware of our guidelines here, should we rather increase to 4.9.0?

Reply:
It seems to be a minor version change. I use https://semver.org/ as reference

Reply:
to be fair, I have no idea of Blacklane guidlines, but as @thiagopio , I was thinking maybe a minor to 4.9.0?

Reply:
Changed.